src/transformers/models/auto/modeling_auto.py

```diff
@@ -475,6 +477,7 @@
 MODEL_FOR_VIDEO_CLASSIFICATION_MAPPING_NAMES = OrderedDict(
     [
         ("timesformer", "TimesformerForVideoClassification"),
+        ("tvlt", "TvltForVideoClassification"),
```

Comment:
This model shouldn't probably be added here since its API is very different from the existing models in this mapping. You can add this model to the list of non-auto configured models in utils/check_repo.py